extensions/s3/build.gradle

```diff
@@ -12,9 +12,10 @@ dependencies {
     implementation project(':Util')
     implementation project(':Configuration')
 
-    implementation platform('software.amazon.awssdk:bom:2.21.43')
+    implementation platform('software.amazon.awssdk:bom:2.23.19')
     implementation 'software.amazon.awssdk:s3'
     implementation 'software.amazon.awssdk:aws-crt-client'
+    //implementation 'software.amazon.awssdk:netty-nio-client'
```

Comment:
Maybe we should benchmark?

Reply:
We need better way to benchmark outstanding branches...

Reply:
Stan has been working on this, but I don't think he has s3 reading benchmarks yet.